docs/root/intro/arch_overview/http/http_connection_management.rst

```diff
@@ -257,9 +277,11 @@ configuration.
 HTTP header map settings
 ------------------------
 
-Envoy maintains the insertion order of headers (and pseudo headers that begin with ":") in the
+Envoy maintains the insertion order of headers (and pseudo headers that begin with ``:``) in the
 HTTP header map using a linked list data-structure, which is very fast when the number of headers
-is small. In addition it can use a map data-structure to ensure fast access to the various headers.
+is small.
+
+In addition it can use a map data-structure to ensure fast access to the various headers.
 The map will be used once the number of headers in a HTTP request/response reaches the value of the
 ``envoy.http.headermap.lazy_map_min_size`` runtime feature. The default threshold value is set to
```

Comment:
not sure what the status of this is - its marked as deprecated and there has been issues with using it historically (cf #13504) so im wondering if we should just remove this documentation - i dont see any alternative for it

Reply:
Yeah I think for now we should probably keep it? cc @alyssawilk

Reply:
cool - ill switch this to an rst link so if/when it does get removed it will break the docs here

Reply:
please remove it - it now a no-op which I'll clean up